packages/react-renderer-demo/src/pages/hooks/use-form-api.md

```diff
@@ -26,6 +26,12 @@ This hook returns object containing following information:
 
 If you want to render fields from a component (`tabs`, `subform`, etc.) you can use `renderForm(fields)` function.
 
+## schema
+
+*object*
+
+The schema prop of FormRenderer.
```

Comment:
Can we get a link to schema here? 

https://data-driven-forms.org/components/renderer#schema